modules/apps/29-fee/keeper/escrow_test.go

```diff
@@ -225,8 +275,34 @@ func (suite *KeeperTestSuite) TestDistributePacketFeesOnTimeout() {
 				balance := suite.chainA.GetSimApp().BankKeeper.GetBalance(suite.chainA.GetContext(), timeoutRelayer, sdk.DefaultBondDenom)
 				suite.Require().Equal(expectedTimeoutAccBal, balance)
 
-				// check if the refund acc has been refunded the recv/ack fees
-				expectedRefundAccBal := refundAccBal.Add(defaultAckFee[0]).Add(defaultAckFee[0]).Add(defaultRecvFee[0]).Add(defaultRecvFee[0])
+				// check if the refund amount is zero
+				expectedRefundAccBal := refundAccBal
+				balance = suite.chainA.GetSimApp().BankKeeper.GetBalance(suite.chainA.GetContext(), refundAcc, sdk.DefaultBondDenom)
+				suite.Require().Equal(expectedRefundAccBal, balance)
+
+				// check the module acc wallet is now empty
+				balance = suite.chainA.GetSimApp().BankKeeper.GetBalance(suite.chainA.GetContext(), suite.chainA.GetSimApp().IBCFeeKeeper.GetFeeModuleAddress(), sdk.DefaultBondDenom)
+				suite.Require().Equal(sdk.NewCoin(sdk.DefaultBondDenom, sdkmath.NewInt(0)), balance)
+			},
+		},
+		{
+			"success: some refund amount",
+			func() {
+				// set the recv + ack fee to be greater than timeout fee so that the refund amount is non-zero
+				fee = types.NewFee(sdk.NewCoins(sdk.NewCoin(sdk.DefaultBondDenom, sdkmath.NewInt(200))), defaultAckFee, defaultTimeoutFee)
```

Comment:
do we need a new `fee` testing var here? can't we just use continue using `packetFee` and a const for fee?

Reply:
It was defined previously. I didn't wanna remove it and break tests.